### ISSUE TITLE
Undefined index: value (When using select_from_array)

### ISSUE THREAD

**Issue by madd86** [Laravel-Backpack/CRUD#312]
PHP: 7.0
Ubuntu 16.04 (regular LAMP)
Laravel: 5.3

I have two projects, backpack-demo and larafin (my own).

This is the code:

```
$this->crud->addField([ 
		'name' => 'isPaid',
		'label' => 'Paid',
		'type' => 'select_from_array',
	'options' => ['Yes' => 'Yes', 'No' => 'No'],
	'allows_null' => false,
]);
```

Works awesome on backpack-demo.

On my own project on the other hand I get this: 

`Undefined index: value (View: /var/www/html/larafin/vendor/backpack/crud/src/resources/views/fields/select_from_array.blade.php) (View: /var/www/html/larafin/vendor/backpack/crud/src/resources/views/fields/select_from_array.blade.php) (View: /var/www/html/larafin/vendor/backpack/crud/src/resources/views/fields/select_from_array.blade.php)`

I wonder why.

**Comment by OwenMelbz** [comment]
I came across this the other day, a quick fix is to add an empty value param to your array going into the addField

**Comment by madd86** [comment]
Huh, what you know, it works. Thanks.

I wonder why this is going on though. Maybe because backpack-demo uses 5.2 and I'm using 5.3?

**Comment by OwenMelbz** [comment]
Something funky is going on behind the scenes where on the "create" the value param doesn't exist, which causes some issues. It should exist I think but maybe @tabacitu can shed some light

**Comment by tabacitu** [comment]
Huh. Don't know, but I'll investigate and come back.

**Comment by tabacitu** [comment]
Fixed in ```3.1.49``` (https://github.com/Laravel-Backpack/CRUD/pull/341) thanks to [Chris Thompson](https://github.com/ChrisThompsonTLDR), using exactly @IvanBernatovic 's method. 

Thanks guys, cheers!

P.S. I'll go ahead and close this issue so we don't clog the section. Feel free to open it again if you encounter problems or comment if you have follow-ups.

**Comment by gabrieladeniji** [comment]
http://bot.tigereum.com/
